### ISSUE TITLE
Inaccessibility: A11Y Project team member cards violate WCAG 2.5.3: Label in Name

### ISSUE THREAD

**Issue by mxmason** [a11yproject/a11yproject.com#1264]
## Summary
The `Website` links on the A11Y project team page have the visible label `Website`, but their accessible names start with other words. The `./src/_includes/person.njk` template should be updated with expected link text.

1. Navigate to https://www.a11yproject.com/team/
2. Start VoiceOver (or NVDA, or similar)
3. Navigate to Scott Vinkle's card
4. Read the card contents with the screen reader.
5. Observe that accessible name of the `Website` link is `SCOTT VINKLE'S WEBSITE`

## Expected result
The accessible name of the `Website` link should either 
- **exactly match** its visible text or
- **begin with** its visible text

## Actual result
The accessible name of the `Website` link neither matches its visible text nor begins with its visible text.

<img src="https://user-images.githubusercontent.com/13525251/116596520-4c83ae80-a8d9-11eb-84c1-cbf2e6af32c1.png" alt="https://user-images.githubusercontent.com/13525251/116596520-4c83ae80-a8d9-11eb-84c1-cbf2e6af32c1.png" class="transparent shrinkToFit" width="600" height="365" alt="Screenshot with tooltip showing DOM and accessibility tree info for Scott Vinkle's Website link">
This screenshot uses Edge browser's dev tools to illustrate the mismatch between the visible text and the accessible name of the link to Scott Vinkle's website. 

## Additional Information
- [WCAG Label In Name success criterion](https://www.w3.org/TR/WCAG21/#label-in-name)
- [Understanding document for Label In Name](https://www.w3.org/WAI/WCAG21/Understanding/label-in-name.html)

**Comment by ericwbailey** [comment]
I accidentally pushed to `main` (sorry!), but I believe I have this addressed via https://github.com/a11yproject/a11yproject.com/commit/9b5889eea5076fb81651ba50ed00be50da25e5bf, https://github.com/a11yproject/a11yproject.com/commit/c6a761f8ad80bf002fddd03fc173f5fe7f17b2ed, and https://github.com/a11yproject/a11yproject.com/commit/4f09b46d8ca6c0061eef1836eef6c8d75d260149. I'm going to trigger a build in Netlify for you to verify. 

In the meantime, here is a screenshot of how it looks locally:

<img width="948" alt="Nat Tarnoff's author card, showing their website, bio, and a list of 3 posts they've written." src="https://user-images.githubusercontent.com/634191/116950671-b7274800-ac53-11eb-9869-eb1743eb4cc0.png">

Also, as a followup, we'll probably want to:

- Rig up some logic for what to do when an author's last name ends with an "s" (ex: Andrew Nevins's website).
- Also have some logic to change "…and maybe you!"'s link to be a little less janky.
- Protect `main` to prevent people from pushing directly to it.

**Comment by mxmason** [comment]
Ooh, thanks, @ericwbailey 

1. Issue seems closed to me.
2. There's no ✨ rule ✨ about showing possession with names ending in `s`, and we can leave the placement of the apostrophe up to our own stylistic preferences. I prefer to place the apostrophe after a noun only when the noun is plural (`My sisters' bookstore`), but I am happy to adopt whatever style guide you prefer.

**Comment by ericwbailey** [comment]
Agree with you on the apostrophe placement. Closing.
